jobs/config.json

```diff
@@ -8857,16 +8917,6 @@
       "sig-node"
     ]
   },
-  "ci-kubernetes-node-kubelet-non-cri-1.6": {
```

Comment:
This job is not needed anymore?

Reply:
@yujuhong I remember this job is to verify the old behavior when we switched to cri right? Do we still care about non-cri? Since we are not testing against newer branches.

Reply:
We care about non-cri behavior in the 1.6 branch. As long as we still accept patches to 1.6, we should keep this.

Reply:
ack, thanks

Reply:
added back